code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -443,6 +463,7 @@
 /obj/item/gun_upgrade/trigger/better
 	name = "Refined trigger"
 	desc = "This trigger seems to be made of durable alloys and cut to the precision of milimeters."
+	matter = list(MATERIAL_STEEL = 2, MATERIAL_PLASTEEL = 3, MATERIAL_DIAMOND = 1)
```

Comment:
Inbuild mods from techno lathes are especially a problem if they give mats as these now give valuable mats and can be done theoretically infinitely as inbuilt designs on the lathe don't have license points